src/request/index.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+const BASE_URL=process.env.REACT_APP_BACKEND_BASE_URL
+
+export default axios.create({
```

Comment:
add a .env.example file to this projects, that contains the environment variable setup, so i can pull and test it